sphinx/developers/java-library.rst

```diff
@@ -98,11 +98,11 @@ The complete list of third-party dependencies for `formats-gpl.jar` is as follow
     * - `Commons Logging v1.2 <http://commons.apache.org/logging/>`_
       - commons-logging:commons-logging:1.2
       - `Apache License v2.0`_
-    * - `NetCDF-Java Library v5.3.3 <https://docs.unidata.ucar.edu/netcdf-java/5.3/userguide/index.html>`_
-      - edu.ucar:cdm:5.3.3
-      - `MIT-Style License`_
-    * - `Joda time v2.2 <https://github.com/JodaOrg/joda-time>`_
-      - joda-time:joda-time:2.2
+    * - `NetCDF-Java Library v5.6.0 <https://docs.unidata.ucar.edu/netcdf-java/5.3/userguide/index.html>`_
```

Comment:
Update URL to https://docs.unidata.ucar.edu/netcdf-java/5.6/userguide/index.html or similar, since the current one still references 5.3?